apps/console/src/features/identity-providers/data/identity-provider-templates/templates/saml-identity-provider/saml-idp-wizard-help.tsx

```diff
@@ -17,47 +17,51 @@
  */
 
 import { TestableComponentInterface } from "@wso2is/core/models";
-import { Heading } from "@wso2is/react-components";
+import { Code, Heading } from "@wso2is/react-components";
 import React, { FunctionComponent, ReactElement } from "react";
+import { useSelector } from "react-redux";
 import { Divider } from "semantic-ui-react";
+import { AppState, ConfigReducerStateInterface } from "../../../../../core";
 
 type Props = TestableComponentInterface;
 
 const SamlIDPWizardHelp: FunctionComponent<Props> = (props: Props): ReactElement => {
 
     const { [ "data-testid" ]: testId } = props;
 
+    const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
+
     return (
         <div data-testid={ testId }>
-            <Heading as="h5">Service provider entity id</Heading>
+            <Heading as="h5">Service provider entity ID</Heading>
             <p>
-                A globally unique name for IDPs under Asgardeo. This can be any value but when you configure a service
-                provider in the external IDP you should give the same value as the Service Provider Entity ID.
+                This value will be used as the <Code>&lt;saml2:Issuer&gt;</Code> in the SAML requests initiated from
```

Comment:
Text should be retrieved from the i18n module. Hence please move these to the language files.